src/app/components/Calendar/CustomCalendar.css

```diff
@@ -0,0 +1,126 @@
+.react-calendar {
+  width: 320px;
+  max-width: 100%;
+  background: white;
+  border: none;
+}
+
+/* Adapt Nav Bar // Is there a way to overwrite the inline style of the navigation that sets display=flex to grid, other that using !important? */
+.react-calendar__navigation {
+  display: grid !important;
+  grid-template-columns: 1% 9% 80% 9% 1%;
+  justify-items: center;
+}
+
+/* Month */
+.react-calendar__navigation__label__labelText--from {
+  color: #f9f2ff;
+  font-size: 2rem;
+}
+
+/* Arrow to next month */
+.react-calendar__navigation__next-button {
+  color: #8312a5;
+  font-size: 2.5rem;
+}
+
+/* Arrow to previous month */
+.react-calendar__navigation__prev-button {
+  color: #8312a5;
+  font-size: 2.5rem;
+}
+
+/* Remove arrow to same month, next year */
+.react-calendar__navigation__next2-button {
+  color: transparent;
+  padding: 0;
+}
+
+/* Remove arrow to same month, previous year */
+.react-calendar__navigation__prev2-button {
+  color: transparent;
+  padding: 0;
+}
+
+/* Mo-So */
+.react-calendar__month-view__weekdays {
+  font-family: var(--font-family-primary);
+  font-weight: 300;
+  color: #8312a5;
+  font-size: 0.9rem;
+  text-align: center;
+  text-transform: none;
+}
+
+/* einzelner Tag // How to remove dotted underline below Mo, Di Mi, Do, Fr, Sa, So? Text-decoration: none does not seem to work. */
+.react-calendar__month-view__weekdays__weekday {
+  text-decoration: none;
+}
```

Comment:
Could the dotted line be a `border`?

Reply:
Mo, Di, Mi, etc are in an <abbr> tag, so I can remove the dots by using that tag